### ISSUE TITLE
Remove aria-owns attribute

### ISSUE THREAD

**Issue by LJWatson** [nomensa/jquery.hide-show#12]
Believe the aria-owns attribute should be removed from the <button> element. A show/hide widget like this doesn't need a parent/child relationship between the control and the content (aria-controls is fine) [1]. Consequently it's causing issues with Jaws16 (the only AT to support aria-owns at present).

<div class="js-hide-show">
<button class="js-hide-show-btn js-hide-show-btn--expanded" id="btn-control-0" aria-owns="content-0" aria-controls="content-0">Hide Content</button>
...
</div>

[1] http://www.w3.org/TR/wai-aria/states_and_properties#aria-owns
